RELEASE_NOTES.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.0] - 2024-02-14
```

Comment:
We're just adding a method here, so for 0.x what I usually do is treat the 3rd number as the minor release and the 2nd number as a major release. So 0.11.7

Reply:
MAJOR.MINOR.PATCH based on https://semver.org/. 
So the 2nd number as a minor release and the 3rd number as the patch.

Reply:
Doesn't apply for 0.x versions

Reply:
:+1: